api-types/index.ts

```diff
@@ -70,6 +70,12 @@ export type DesktopAPI = {
 
     focusPopout: () => void;
 
+    openThread: (threadID: string) => void;
```

Comment:
Can we rename these so that we know they're specific to calls?
Something like `openThreadForCalls`.
I know it's a bit "smurf naming" but it's important in this context to make sure the API is only used for that.

Reply:
Sure. This functionality was generic enough that I thought it could be used by something else in the future, but we can rename it for now since Calls is currently the only user.

Reply:
@devinbinnie Done